libraries/core-react/src/components/Progress/Dots/DotProgress.test.tsx

```diff
@@ -36,4 +36,14 @@ describe('DotProgress', () => {
     const progressbar = screen.getByRole('progressbar')
     expect(progressbar).toHaveStyleRule('position', 'absolute')
   })
+  it('has correct default width ', () => {
+    const { container } = render(<StyledProgress />)
+    const progress = container.firstChild
+    expect(progress).toHaveAttribute('width', '32')
+  })
+  it('has correct width when size is 64', () => {
+    const { container } = render(<StyledProgress size={64} />)
+    const progress = container.firstChild
+    expect(progress).toHaveAttribute('width', '64')
+  })
```

Comment:
Assuming that we shouldn't have knowledge of implementation details, would it be better to test for height as well? 🤔

Reply:
Its 1/4 of height, which we could test.. but then again, its kinda implementation detail and kinda not 🤔 @wenche

Reply:
Yeah... Maybe it could be more of a "has height set" approach to make sure it's a circle

Reply:
Updated tests @wenche

Reply:
👍